.azure/pipelines/ci.yml

```diff
@@ -632,42 +632,43 @@ extends:
-        - ${{ if in(variables['Build.Reason'], 'PullRequest', 'Manual') }}:
-          - template: .azure/pipelines/jobs/default-build.yml@self
-            parameters:
-              jobName: Local_Windows
-              jobDisplayName: 'Test: Windows local development validation'
-              agentOs: Windows
-              isAzDOTestingJob: true
-              timeoutInMinutes: 240
-              steps:
-              - script: git submodule update --init
-                displayName: Update submodules
-              - script: ./restore.cmd
-                displayName: Run restore.cmd
-              - script: ./eng/build.cmd -all -noBuildJava -noBuildNodeJS
-                displayName: Build (No NodeJS)
-              - script: npm run build
-                displayName: Build JS
-              - script: ./eng/build.cmd -all -noBuildJava -pack -c Debug
-                displayName: Build (Debug)
-              - script: ./eng/build.cmd -all -noBuildJava -pack -c Release
-                displayName: Build (Release)
-              - script: ./src/ProjectTemplates/build.cmd
-                        -test
-                        -NoRestore
-                        -NoBuild
-                        -NoBuildDeps
-                        -configuration Release
-                displayName: Run project template tests
-              - powershell: . ./activate.ps1; ./src/ProjectTemplates/scripts/Run-BlazorWeb-Locally.ps1 -Verbose
-                displayName: Run Blazor web app test script
-
-              artifacts:
-              - name: Local_Windows_x64_Logs_Attempt_$(System.JobAttempt)
-                path: artifacts/log/
-                publishOnError: true
-                includeForks: true
+        - template: .azure/pipelines/jobs/default-build.yml@self
+          parameters:
+            jobName: Local_Windows
+            jobDisplayName: 'Test: Windows local development validation'
+            agentOs: Windows
+            isAzDOTestingJob: true
+            timeoutInMinutes: 240
+            steps:
+            - script: git submodule update --init
+              displayName: Update submodules
+            - script: ./restore.cmd
+              displayName: Run restore.cmd
+            - powershell: ./eng/build.ps1 -all -noBuildJava -noBuildNodeJS "-WarnAsError:false"
+              displayName: Build (No NodeJS)
+            - script: npm run build
+              displayName: Build JS
+            - script: ./eng/build.cmd -all -noBuildJava -pack -c Debug
+              displayName: Build (Debug)
+            - script: ./eng/build.cmd -all -noBuildJava -pack -c Release
+              displayName: Build (Release)
+            - script: ./src/ProjectTemplates/build.cmd
+                      -test
+                      -NoRestore
+                      -NoBuild
+                      -NoBuildDeps
+                      -configuration Release
+                      -bl
+                      /p:ContinuousIntegrationBuild=true
```

Comment:
The project template tests also run in other pipelines, not just the local development validation one. Is there a reason why we need to use this pipeline in particular to generate CG alerts? I only ask because this pipeline is primarily intended as a way to make sure developers can test changes locally, and it would be unusual to pass `/p:ContinuousIntegrationBuild=true` when doing so.

Reply:
Sounds good - would it work to introduce a new property that gets specified for one of the other existing pipelines?

Reply:
Yeah, let's go with another property - suggestion on which other pipeline to use? I just noticed that the Windows test job skips the template tests: https://github.com/dotnet/aspnetcore/blob/bb7c3930365d7471f6a73b64888ddad92a394dd3/.azure/pipelines/ci.yml#L548

Reply:
Oh, interesting. Maybe Ubuntu x64? No preference really 🙂